estate/models/estate_property.py

```diff
@@ -36,3 +37,46 @@ class EstateProperty(models.Model):
         copy=False,
         default="new",
     )
+    property_type = fields.Many2one("estate.property.type")
+    buyer = fields.Many2one("estate.property.type", copy=False)
```

Comment:
this is related to res.partner model

```suggestion
    buyer_id = fields.Many2one("res.partner", copy=False)
```